src/index.html

```diff
@@ -4,9 +4,304 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__links">
+        <a href="#homePage" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav>
+          <ul class="header__list">
+            <li class="header__link">
+              <a href="#Apple" class="header__link-a is-active">APPLE</a>
+            </li>
+            <li class="header__link">
+              <a href="#Samsung" class="header__link-a">SAMSUNG</a>
+            </li>
+            <li class="header__link">
+              <a href="#phones" class="header__link-a">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+            </li>
+            <li class="header__link" data-qa="nav-hover">
+              <a href="#laptops" class="header__link-a">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+            </li>
+            <li class="header__link">
+              <a href="#gadgets" class="header__link-a">ГАДЖЕТЫ</a>
+            </li>
+            <li class="header__link">
+              <a href="#tablet" class="header__link-a">ПЛАНШЕТЫ</a>
+            </li>
+            <li class="header__link">
+              <a href="#photo" class="header__link-a">ФОТО</a>
+            </li>
+            <li class="header__link">
+              <a href="#video" class="header__link-a">ВИДЕО</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="catalog">
+      <section class="card catalog__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+        <div class="card__caption">
+          Моноблок APPLE A1419 iMac 27'' Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card__kod">
```

Comment:
```suggestion
        <div class="card__code">
```
Я думаю, так название класса будет понятнее :)